astropy/stats/sigma_clipping.py

```diff
@@ -6,6 +6,14 @@
 import warnings
 from ..utils.exceptions import AstropyDeprecationWarning, AstropyUserWarning
 
+try:
+    from scipy.stats import sigmaclip
+    HAS_SCIPY=True
+except:
```

Comment:
It's good practice to name the exception (now ``ImportError``) that we are expecting so it still fails if something unexpected happens.